### PR TITLE
fix read archived notifications

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/br5hHnmh/7155-qa-new-inbox-already-read-notifications-are-not-being-properly-shown

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA] [new inbox] already read notifications are not being properly shown](https://trello.com/c/br5hHnmh/7155-qa-new-inbox-already-read-notifications-are-not-being-properly-shown)

File: src/app/inbox/components/notification/notification.component.scss-theme.scss [hunk 1]
```diff
@@ -11,10 +11,10 @@
   }
   :host.archived .header {
     background-color: map-get($background, background);
-    color: $orcid-dark-secondary-text;
+    color: $text-dark-mid !important;
```
Comment (leomendoza123) [inline src/app/inbox/components/notification/notification.component.scss-theme.scss:14]: Please use the foreground palette do add these colors

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/inbox/components/notification/notification.component.scss-theme.scss [hunk 2]
```diff
@@ -11,10 +11,10 @@
   }
   :host.archived .header {
     background-color: map-get($background, background);
-    color: $text-dark-mid !important;
+    color: map-get($foreground, 'text-dark-mid') !important;
```
Comment (leomendoza123) [inline src/app/inbox/components/notification/notification.component.scss-theme.scss:14]: This won't work unless is added to the foreground palette 
https://github.com/ORCID/orcid-angular/blob/bde83b34b66386012920018cc445e0573c83e1d8/src/assets/scss/material.light-theme.scss#L42-L53

Am i wrong?

Comment (leomendoza123) [inline src/app/inbox/components/notification/notification.component.scss-theme.scss:14]: This won't work unless is added to the foreground palette 
https://github.com/ORCID/orcid-angular/blob/bde83b34b66386012920018cc445e0573c83e1d8/src/assets/scss/material.light-theme.scss#L42-L53

Am i wrong?

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/inbox/components/notification/notification.component.scss-theme.scss (hunk 2 above)
Comment (DanielPalafox) [reply src/app/inbox/components/notification/notification.component.scss-theme.scss:14]: Could you double-check it is already there @leomendoza123

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**